src/private/xml_parser.cpp

```diff
@@ -193,6 +193,13 @@ namespace adm {
       addOptionalReferences<AudioPackFormatId>(node, "audioPackFormatIDRef", audioObject, objectPackFormatRefs_, &parseAudioPackFormatId);
       addOptionalReferences<AudioTrackUidId>(node, "audioTrackUIDRef", audioObject, objectTrackUidRefs_, &parseAudioTrackUidId);
       setOptionalElement<AudioObjectInteraction>(node, "audioObjectInteraction", audioObject, &parseAudioObjectInteraction);
+
+      if(guessCartesianFlag(node, "positionOffset") == false) {
```

Comment:
It's a personal preference, but I would find this easier to parse if it was just (`if(guessCartesianFlag(node, "positionOffset")) {` with the cases swapped around.